### DISCUSSION TITLE
How often are changes to the API being deployed? Is there a place that has a changelog?

### DISCUSSION THREAD

**Discussion by jnorris441** [supabase/supabase#5836] [Questions] [answered]
In the few weeks I've been using Supabase I've had a few queries work perfectly fine, and then stop working a few days later.

Is there a place to track what changes are being made to the API so we don't get caught out by queries that stop working?

**Comment by steve-chavez** [comment, marked as answer]
@jnorris441 Can you confirm your broken queries are of the same type as the ones reported in https://github.com/supabase/supabase/discussions/5801?

**Reply by jnorris441** [reply]
@steve-chavez Yes that was one of them. Sounds like it will be better going forward 👍
